docs/src/modules/components/branding/Page.js

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import SearchAppBar from 'docs/src/modules/components/branding/SearchAppBar';
+import Footer from 'docs/src/modules/components/branding/Footer';
+
+const theme = createMuiTheme({
+  palette: {
+    text: {
+      primary: '#001E3C',
+    },
+    primary: {
+      main: '#007FFF',
+    },
+    secondary: {
+      main: '#001E3C',
+    },
+    background: {
+      default: '#FFF',
+    },
+    vividBlue: '#00C8FF',
+    greyF3: '#F3F6F9',
+    greyEAE: '#EAEEF3',
+    grey5A: '#5A6978',
+    azure: '#007FFF',
+  },
+  typography: {
+    fontFamily: [
+      'Inter',
+      // system-font
+      '-apple-system',
+      'BlinkMacSystemFont',
+      '"Segoe UI"',
+      'Roboto',
+      '"Helvetica Neue"',
+      'Arial',
+      'sans-serif',
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"',
+    ].join(','),
+    h1: {
+      fontSize: 72,
+      fontWeight: 700,
+      letterSpacing: '-2px',
+      lineHeight: 1.111,
+    },
+    h2: {
+      fontSize: 52,
+      fontWeight: 700,
+      letterSpacing: '-1.5px',
+      lineHeight: 1.154,
+    },
+    h3: {
+      fontSize: 36,
+      fontWeight: 700,
+      letterSpacing: '-1px',
+      lineHeight: 1.222,
+    },
+    h4: {
+      fontSize: 24,
+      fontWeight: 700,
+      letterSpacing: '-0.5px',
+      lineHeight: 1.25,
+    },
+  },
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 960,
+      lg: 1440,
+      xl: 1920,
+    },
+  },
+  components: {
+    MuiButton: {
+      defaultProps: {
+        disableRipple: true,
```

Comment:
https://next.material-ui.com/components/buttons/#contained-buttons

```suggestion
        disableRipple: true,
        disableElevation: true,
```